src/main/kotlin/com/github/mdashl/rankedhelper/command/WdrCommand.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.mdashl.rankedhelper.command
```

Comment:
Rename package back to `commands`
```suggestion
package com.github.mdashl.rankedhelper.commands
```